pytorch/wavenet.py

```diff
@@ -123,10 +120,13 @@ def forward(self, forward_input):
        
         forward_input = self.embed(forward_input.long())
         forward_input = forward_input.transpose(1, 2)
-        
+       
+        cond_acts = self.cond_layers(cond_input)
         for i in range(self.n_layers):
             in_act = self.dilate_layers[i](forward_input)
-            in_act = in_act + self.cond_layers[i](cond_input)
+            start = i*2*self.n_residual_channels
```

Comment:
I think if you do ``cond_acts.view(batch_size, num_layers, block_hidden_size * 2, signal_length)`` you then do not need to compute ``start`` and end``; rather do something like ``cond_acts[:,layer (1, 2, 3..),:,:]``.

Reply:
Changed.